cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -282,24 +281,15 @@ func (guarantor *engineExistenceGuarantor) getRunningAndCLIEngineVersions() (*se
 
 func (guarantor *engineExistenceGuarantor) ensureCentralizedLogsComponentsAreRunning(ctx context.Context, shouldForceContainerRestart bool) error {
 
-	logsDatabase, err := guarantor.kurtosisBackend.GetLogsDatabase(ctx)
+	logsAggregator, err := guarantor.kurtosisBackend.GetLogsAggregator(ctx)
 	if err != nil {
 		return stacktrace.Propagate(err, "An error occurred getting the logs database")
 	}
-	isThereLogsDatabase := logsDatabase != nil
-	isThereNotRunningLogsDatabase := isThereLogsDatabase && logsDatabase.GetStatus() != container_status.ContainerStatus_Running
+	isThereLogAggregator := logsAggregator != nil
 
-	//Destroy the logs database if caller requested it or if the container is not running
-	if shouldForceContainerRestart || isThereNotRunningLogsDatabase {
-		if err = guarantor.kurtosisBackend.DestroyLogsDatabase(ctx); err != nil {
-			return stacktrace.Propagate(err, "An error occurred destroying the logs database")
-		}
-		isThereLogsDatabase = false
-	}
-
-	if !isThereLogsDatabase {
-		if _, err := guarantor.kurtosisBackend.CreateLogsDatabase(ctx, defaultHttpLogsDatabasePortNum); err != nil {
-			return stacktrace.Propagate(err, "An error occurred creating the logs database with HTTP port number '%v'", defaultHttpLogsDatabasePortNum)
+	if !isThereLogAggregator {
```

Comment:
So, if the log aggregator is stopped we don't try to restart it anymore?

Reply:
yes we should. added that.